cmd/generate.go

```diff
@@ -192,6 +195,9 @@ func createProject(sourcePath string) (*AtlantisProject, error) {
 			WhenModified: relativeDependencies,
 		},
 	}
+	if createWorkspace {
+		project.Workspace = strings.ReplaceAll(project.Dir, "/", "_")
```

Comment:
Does a workspace named "." work as a full workspace name with atlantis? Or should we also replace the "." character in the relative path with some other character?

Reply:
seems like the `.` isn't valid as per the doc: https://www.terraform.io/docs/cloud/workspaces/naming.html, I have modified the code to accomodate that.